cli/src/commands/content/updateChannelCensorshipStatus.ts

```diff
@@ -0,0 +1,77 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import chalk from 'chalk'
+import ExitCodes from '../../ExitCodes'
+
+export default class UpdateChannelCensorshipStatusCommand extends ContentDirectoryCommandBase {
+  static description = 'Update Channel censorship status (Active/Inactive).'
+  static flags = {
+    context: ContentDirectoryCommandBase.contextFlag,
+  }
+  static args = [
+    {
+      name: 'id',
+      required: false,
+      description: 'ID of the Channel',
+    },
+    {
+      name: 'status',
+      required: false,
+      description: 'New status of the channel (1 - active, 0 - inactive)',
+    },
+    {
+      name: 'rationale',
+      required: false,
+      description: 'rationale',
+    },
+  ]
+
+  async run() {
+    let { context } = this.parse(UpdateChannelCensorshipStatusCommand).flags
+
+    let { id, status, rationale } = this.parse(UpdateChannelCensorshipStatusCommand).args
+
+    if (!context) {
+      context = await this.promptForContext()
+    }
+
+    const currentAccount = await this.getRequiredSelectedAccount()
+    await this.requestAccountDecoding(currentAccount)
+
+    const actor = await this.getActor(context)
+
+    if (id === undefined) {
+      id = await this.promptForChannel()
```

Comment:
`promptForChannel` return `Channel` struct (not an id), which causes this command to fail.
I encourage you to strongly-type the return value (especially when it's `any` by default) to avoid mistakes like that

Reply:
Done, thank you for the suggestion. https://github.com/Joystream/joystream/pull/2280/commits/c2e27791c688f13158d745920ce869fbaa6ef556